presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/rowsRangeRankFunctions.sql

```diff
@@ -0,0 +1,9 @@
+-- database: presto; groups: window;
```

Comment:
What's so special about this test?
Isn't it covered already by `rowsRangeDifferentPartition.sql`?